components/admin_console/schema_admin_settings.jsx

```diff
@@ -666,13 +688,144 @@ export default class SchemaAdminSettings extends AdminSettings {
         );
     }
 
+    closeTooltip = () => {
```

Comment:
All this is copied from AdminSettings class to remove the relationship (parent/child)